### ISSUE TITLE
Please document how to use lists

### ISSUE THREAD

**Issue by tvincent2** [ruuda/pris#26]
Hi!

The current example about lists describes how to declare lists, but not how they can be used. Please show we can use them.

Thanks!
Thomas

**Comment by ruuda** [comment]
You are totally right, I should write better documentation. In the mean time, [there is the polygon example](https://github.com/ruuda/pris/blob/ea0d952f23e5e7eb4d9392fbbc14069b329b6e1a/examples/polygon.pris) and [the roundrect example](https://github.com/ruuda/pris/blob/ea0d952f23e5e7eb4d9392fbbc14069b329b6e1a/examples/roundrect.pris).
